static/app/views/performance/landing/metricsDataSwitcher.spec.tsx

```diff
@@ -257,8 +259,7 @@ describe('Performance > Landing > MetricsDataSwitcher', function () {
       nullCount: 0,
       unparamCount: 1,
     });
-    const platformWithDocs = 'javascript.react';
-    const project = TestStubs.Project({platform: platformWithDocs});
+    const project = ProjectFixture({platform: 'javascript-react'});
```

Comment:
it was `javascript.react` before.

Reply:
https://github.com/getsentry/sentry/blob/3b1feab826697c3efed6dd10a650e5a9b898ba99/static/app/types/project.tsx#L198